DebugGUI/src/DebugGUI.h

```diff
@@ -11,6 +11,7 @@
 #define FRAMEWORK_DEBUGGUI_H
 
 #include <functional>
+#include <ostream>
```

Comment:
```suggestion
#include <iosfwd>
```

Reply:
I need ostream for the stream operator

Reply:
Don't you just need the implementation in the .h?